ziggurat-core-crawler/src/summary.rs

```diff
@@ -20,9 +22,9 @@ pub struct NetworkSummary {
     /// Crawler's runtime.
     pub crawler_runtime: Duration,
     /// Addresses of good nodes.
-    pub node_ips: Vec<String>,
+    pub node_addrs: Vec<SocketAddr>,
     /// Unidirected connections graph.
-    pub indices: Vec<Vec<usize>>,
+    pub node_indices: NodeIndices,
```

Comment:
The same as above: maybe `nodes_indices`.

Reply:
Here, I'm merely being consistent with the typename.  Which, sure, I guess could be `NodesIndices`, but I chose to make it somewhat simplified.

Reply:
hmm, thinking about this. I think `GraphIndices` would be better here.  `NodeIndices` is weaker.

Reply:
`NodesIndicies` and `nodes_indices` it is, FTW!